smack-experimental/src/main/java/org/jivesoftware/smackx/stanza_content_encryption/element/FromAffixElement.java

```diff
@@ -0,0 +1,50 @@
+/**
+ *
+ * Copyright 2020 Paul Schaub
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jivesoftware.smackx.stanza_content_encryption.element;
+
+import org.jivesoftware.smack.util.Objects;
+
+import org.jxmpp.jid.Jid;
+
+public class FromAffixElement extends JidAffixElement {
+
+    public static final String ELEMENT = "from";
+
+    public FromAffixElement(Jid jid) {
+        super(jid);
+    }
+
+    @Override
+    public String getElementName() {
+        return ELEMENT;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
Similarly, EqualsUtil might be of help.

Reply:
I guess using `EqualsUtil` is probably overkill and would make the code less readable in this case.

Reply:
Not sure, with equals util this code would melt down to

```
return EqualsUtil.equals(this, obj, (e, o) → e.append(getJid(), o.getJid());
```

I think it's worth using `EqualsUtil` even in "simple" cases, bu I leave it to your discretion if you want to use it in those cases. Just marking as unresolved to give you a chance to reconsider.